### PR TITLE
Add 3 second delay after clicking Battle!

### PR DESCRIPTION
**Pull request by acz13**

### CODE REVIEW

**Comment by acz13** [comment]
Fuck my git skills

**Comment by panpawn** [comment]
... won't this make it so for those three seconds, you can't click cancel because the button won't appear...?

**Comment by acz13** [comment]
No? The button doesn't appear when it sends it it appears as soon as you click the button

(check testclient.html if you don't believe me)

File: js/client-mainmenu.js [hunk 1]
```diff
@@ -859,9 +859,12 @@
 			$searchForm.append('<p class="cancel buttonbar"><button name="cancelSearch">Cancel</button></p>');
```
Comment (Zarel) [inline js/client-mainmenu.js]: @panpawn, the button is inserted here, before the searchDelay even starts.

**Comment by Zarel** [comment]
This is good! Can I get you to agree to the MIT license?

Repeat after me: "I hereby license all my contributions to the Pokémon Showdown project, past and future, under the MIT license."

File: js/client-mainmenu.js (hunk 1 above)
Comment (panpawn) [inline js/client-mainmenu.js]: ohh, ok

**Comment by acz13** [comment]
I believe I've done that before #756   
but hey:  
I hereby license all my contributions to the Pokémon Showdown project, past and future, under the MIT license.

**Comment by Zarel** [comment]
Oh, that's you? You don't need to do it again. Do you use the Dev room? Do I know you?

**Comment by acz13** [comment]
>Oh, that's you?

Yeah...?

>Do you use the Dev room?

Sometimes (mostly I hang out in `#showdown` though these days as I don't frequent PS! much

>Do I know you?

I don't think so...?

**Comment by Zarel** [comment]
You should hang out in the Dev room more. I don't use #showdown much because I already have way too many chat clients running on my computer and don't need another. :/

**Comment by acz13** [comment]
Well, my school blocks `http://*.pokemonshowdown.com/*` so I can't really chat for half the day (except on a certain Spanish speaking server...)

**Pull request merged**